app/scripts/views/mixins/signin-mixin.js

```diff
@@ -24,13 +31,15 @@ define(function (require, exports, module) {
      * @return {object} promise
      */
     signIn: function (account, password) {
+      var self = this;
+      self.logEvent(`flow.${this.signInSubmitContext}.submit`);
```

Comment:
btw, all views have a viewName based off of their URL, if one is not already specified on the View's prototype.